WAKU.md

```diff
@@ -0,0 +1,17 @@
+# Waku spec support
+
+*Last updated February 14, 2020*
+
+status-go client of Waku is spec compliant with [Waku spec v0.3](https://specs.vac.dev/waku.html) with the exception of:
+- It doesn't support all the MUST packet codes (!)
+- Currently nodes with higher version don't automatically disconnect if versions are different
+
+It doesn't yet implement the following recommended features:
+- It doesn't apply a timeout to receive Status message
```

Comment:
```suggestion
- It doesn't apply a timeout to receive Status message, nor does it disconnect a peer if it receives another message type before it receives a Status message
```

Reply:
@adambabik

Reply:
Keep it as it is. It does disconnect: https://github.com/status-im/status-go/blob/develop/waku/peer.go#L115-L117.

Reply:
Ack, I was going of

- [ ] 19. When a node does not receive the Status message from a peer, before a configurable timeout, it SHOULD disconnect from that peer.
- [ ] 21. [...] message is received, the node MUST ignore these messages and SHOULD disconnect from that peer.

maybe I misunderstood them? Leaving as is